splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -1925,4 +1904,18 @@ public long getSystablesMinRetentionPeriod() {
     public boolean useTxnAwareCache() {
         return !SpliceClient.isRegionServer;
     }
+
+    public void updateSysConstraintTable(TransactionController tc) throws StandardException {
+        SchemaDescriptor sd = getSystemSchemaDescriptor();
+        tc.elevate("dictionary");
+
+        TableDescriptor td = getTableDescriptor("SYSCONSTRAINTS", sd, tc);
+        if (td != null) {
+            dropAllColumnDescriptors(td.getUUID(), tc);
+            dropTableDescriptor(td, sd, tc);
+        }
+
+        createNaturalNumbersTable(tc);
```

Comment:
This drops SYSCONSTRAINTS table. However, you only want to drop a column from it.

If this is an extra column that's to be ignored, it is OK not to drop it.

Reply:
I'm interested to see the final code here before reviewing this.

Reply:
I've rolled back my changes. The field will not be dropped.